bittensor/_cli/cli_impl.py

```diff
@@ -212,6 +215,11 @@ def help ( self ):
             bittensor.neurons.multitron_server.neuron().run()
 
 
+    def update ( self ):
+        if self.config.no_prompt or self.config.answer == 'Y':
+            os.system(' (cd ~/.bittensor/bittensor/ ; git checkout master ; git pull --ff-only )')
+            os.system('pip install -e ~/.bittensor/bittensor/')
```

Comment:
This is out of scope for this PR, but I see this path in a few places and I don't think it should be hardcoded or we'll have a lot of pain if we change it later (or make it OS-agnostic)

Reply:
Yea, I agree. Right now, bittensor installation is mostly done using the install script. As we move away, this hardcoded path is gonna bite us haha